src/views/htmlcontent/src/html/app.html

```diff
@@ -16,6 +16,8 @@
                         showHeader="true"
                         [resized]="dataSet.resized"
                         (mousedown)="navigateToGrid(i)"
+                        [selectionModel]="selectionModel"
```

Comment:
I'm fairly certain that this file does absolutely nothing and can probably be removed. The hardcoded string in app.component.ts is where the actual HTML is located.